pkg/frontend/querymiddleware/spin_off_subqueries_test.go

```diff
@@ -24,8 +24,6 @@ import (
 )
 
 func TestSubquerySpinOff_Correctness(t *testing.T) {
-	t.Parallel()
-
```

Comment:
Is there a reason why you've removed this? (same applies to other places where `t.Parallel()` has been removed)